locales-id-ID.xml

```diff
@@ -69,13 +69,13 @@
     <term name="page-range-delimiter">–</term>
 
     <!-- ORDINALS -->
-    <term name="ordinal"></term>
-    <term name="ordinal-01"></term>
-    <term name="ordinal-02"></term>
-    <term name="ordinal-03"></term>
-    <term name="ordinal-11"></term>
-    <term name="ordinal-12"></term>
-    <term name="ordinal-13"></term>
+    <term name="ordinal">bilangan bertingkat</term>
```

Comment:
@xbypass, could you use the pencil/edit icon at https://github.com/xbypass/locales/blob/patch-3/locales-id-ID.xml and change the ordinal term translations back from

```xml
    <term name="ordinal">bilangan bertingkat</term>
    <term name="ordinal-01">ke-1</term>
    <term name="ordinal-02">ke-2</term>
    <term name="ordinal-03">ke-3</term>
    <term name="ordinal-11">ke-11</term>
    <term name="ordinal-12">ke-12</term>
    <term name="ordinal-13">ke-13</term>
```

to

```xml
    <term name="ordinal"></term>
    <term name="ordinal-01"></term>
    <term name="ordinal-02"></term>
    <term name="ordinal-03"></term>
    <term name="ordinal-11"></term>
    <term name="ordinal-12"></term>
    <term name="ordinal-13"></term>
```

?